kafka-tools/generate-downloads/README.md

```diff
@@ -41,3 +43,79 @@ $ make run # same make vars as running in env above
 ...
 $ make clean # clean up local files
 ```
+
+## Getting the `INSTANCE_ID`, a worked example
+
+In the support ticket that is missing the `INSTANCE_ID`,the publishing team had provided this URI:
+
+`https://www.ons.gov.uk/datasets/weekly-deaths-local-authority/editions/2022/versions/32`
+
+First decrypt the password for `docdb_login_password` in `dp-setup/ansible/inventories/prod/group_vars` for use below.
+
+Then log into the box in production that has the `mongo` command available:
```

Comment:
could we use the dataset db  password instead of root ?

Reply:
What db password ?

Reply:
what username/password ?

Please provide in slack a command that will log me in in the manner you elude to ..

Reply:
I think those can be found in the secrets for dataset api

Reply:
I've tried a few variations of what has been suggested, so if you would be so kind as to supply a working example ?